requirements.txt

```diff
@@ -1,4 +1,4 @@
 numpy~=1.14
 scipy~=1.0
-scikit-learn>=0.19
-tensorflow>=1.15.2,<2
+scikit-learn>=0.20.0,<0.23.0
```

Comment:
Using 0.19.x and 0.23.x makes the tests fail.  I'm not sure how we didn't catch the 0.19.x incompatibility earlier, but I'd guess that the tests have been running as 0.20+ for a while.